catroid/src/androidTest/java/org/catrobat/catroid/common/firmata/FirmataUtils.java

```diff
@@ -44,7 +51,24 @@ public FirmataUtils(ConnectionDataLogger logger) {
 		this.logger = logger;
 	}
 
-	public FirmataMessage getAnalogMesageData() {
+	public FirmataMessage getDigitalMessageData() {
+
+		int portAndCommand = getNextMessage();
+
+		int port = getPinFromHeader(portAndCommand);
+		int command = getCommandFromHeader(portAndCommand);
+
+		Assert.assertEquals("This is no digital message command", DIGITAL_MESSAGE_COMMAND, command);
```

Comment:
I would not use asserts outside of tests. Those are designed to close the app in case of an unrecoverable state. Please use exceptions and handle them accordingly